services/ui-src/src/components/forms/Form.tsx

```diff
@@ -17,14 +17,18 @@ import {
   hydrateFormFields,
   mapValidationTypesToSchema,
   sortFormErrors,
+  useUser,
```

Comment:
@karla-vm should this be using the new useUserState?

Reply:
ah, yes, `useUserStore`. @keeysnc you might need to merge changes from `main` to pull in the state management stuff.

Reply:
Ah okay, thanks.